robolectric-resources/src/main/java/org/robolectric/RuntimeEnvironment.java

```diff
@@ -41,4 +43,8 @@ public static String getQualifiers() {
     public static void setQualifiers(String newQualifiers) {
```

Comment:
Wouldn't it be better to have

setConfig(Config config)

then the relevant calls would be

RuntimeEnvironmnet.getConfig().getSdk()

and

RuntimeEnvironment.getConfig().getQualifiers()